--- git2nrnversion_h.sh	2018-11-14 10:48:41.000000000 +0100
+++ git2nrnversion_h.sh	2020-11-19 10:00:21.000000000 +0100
@@ -7,7 +7,7 @@

 cd $a

-if git log > /dev/null && test -d .git ; then
+if git log &> /dev/null && test -d .git ; then
         describe="`git describe --tags`"
         branch="`git rev-parse --abbrev-ref HEAD`" # branch name
         modified="`git status -s -uno --porcelain | sed -n '1s/.*/+/p'`" # + if modified
@@ -20,10 +20,9 @@
 elif test -f src/nrnoc/nrnversion.h ; then
         sed -n '1,$p' src/nrnoc/nrnversion.h
 else
-        echo "#define GIT_DATE \"1999-12-31\""
-        echo "#define GIT_BRANCH \"?\""
-        echo "#define GIT_CHANGESET \"?\""
-        echo "#define GIT_DESCRIBE \"?\""
-        exit 1
+        echo "#define GIT_DATE \"Build Time: $(date "+%Y-%m-%d-%H:%M:%S")\""
+        echo "#define GIT_BRANCH \"unknown branch\""
+        echo "#define GIT_CHANGESET \"unknown commit id \""
+        echo "#define GIT_DESCRIBE \"${PROJECT_VERSION}.1\""
 fi
