--- src/nrniv/CMakeLists.txt	2020-11-18 16:51:10.000000000 +0100
+++ src/nrniv/CMakeLists.txt	2020-11-19 11:45:59.000000000 +0100
@@ -139,7 +140,8 @@
 # is different from the contents of nrnversion.h
 # ~~~
 add_custom_target(
-  nrnversion_h ${PROJECT_SOURCE_DIR}/git2nrnversion_h.sh ${PROJECT_SOURCE_DIR} > nrnversion.h.tmp
+  nrnversion_h
+  ${CMAKE_COMMAND} -E env PROJECT_VERSION=${PROJECT_VERSION} ${PROJECT_SOURCE_DIR}/git2nrnversion_h.sh ${PROJECT_SOURCE_DIR} > nrnversion.h.tmp
   COMMAND ${CMAKE_COMMAND} -E copy_if_different nrnversion.h.tmp ${NRN_NRNOC_SRC_DIR}/nrnversion.h
   DEPENDS ${PROJECT_SOURCE_DIR}/git2nrnversion_h.sh)
