--- bin/nrnmech_makefile.in.orig    2018-10-18 12:02:45.000000000 +0200
+++ bin/nrnmech_makefile.in 2018-10-18 12:03:30.000000000 +0200
@@ -46,7 +46,7 @@

 NRNOCLIBS = -L"$(libdir)" -lnrnoc -loc @MEMACSLIB@ \
    $(lnrnmpi) -lscopmath -lsparse13 @READLINE_LIBS@
-NRNIVLIBS = -L"$(libdir)" "$(libdir)/libnrniv.la" -livoc \
+NRNIVLIBS = -L"$(libdir)" -lnrniv -livoc \
    -lneuron_gnu -lmeschach -lsundials \
    $(IVOS_LIB) $(IV_LIBS) $(NJ_LIBS) $(PY_LIBS) $(NRNNI_LIBS) $(PVM_LIBS)
